composer.json

```diff
@@ -8,10 +8,11 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^7.1.3",
+        "php": ">=7.2.0",
```

Comment:
Discussed on Slack, better to use `^7.2` as what we have here implies that this is going to work with PHP 8 off the bat - which it probably wont!